components/glean/build.gradle

```diff
@@ -0,0 +1,201 @@
+// =============================================
+// We're doing something a bit weird here, so
+// READ THIS CAREFULLY before editing this file.
+// =============================================
+//
+// This is a lightly-modified version of the `build.gradle` file from Glean,
+// which you can find at `../external/glean/glean-core/android/build.gradle`.
+//
+// The aim here is to pre-package the Glean Kotlin code and publish it under
+// `org.mozilla.appservices:glean` rather than `org.mozilla.telemetry:glean`,
+// and to make it depend on `org.mozilla.appservices:full-megazord` for the
+// compiled Rust code.
+//
+// We expect consumers that want to use Glean and AppServices together, will
+// use our re-packaged version rather than standalone Glean.
+//
+// In order for this to work without heavy modification to the underlying
+// Glean code, we've duplicated much of the logic from Glean's standard
+// build file, including the use of the glean gradle plugin to generate
+// the standard builtin metrics. As upstream changes happen in Glean's build
+// process we will need to duplicate them in this file.
+
+import groovy.json.JsonOutput
+
+plugins {
+    id "com.jetbrains.python.envs" version "0.0.26"
+}
+
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'kotlin-android-extensions'
+
+File GLEAN_PING_SCHEMA_PATH = file("$rootDir/components/external/glean/glean.1.schema.json")
+
+android {
+    ndkVersion rootProject.ext.build.ndkVersion
+    compileSdkVersion rootProject.ext.build.compileSdkVersion
+
+    defaultConfig {
+        minSdkVersion rootProject.ext.build['minSdkVersion']
+        targetSdkVersion rootProject.ext.build['targetSdkVersion']
+
+        buildConfigField("String", "JNA_LIBRARY_NAME", "\"megazord\"")
```

Comment:
Ref https://github.com/mozilla/glean/pull/1290 for the glean-side change required to make this work.